### DISCUSSION TITLE
Relative date display issue

### DISCUSSION THREAD

**Discussion by RichardD2** [requarks/wiki#4378] [Error / Bug Report]
I have a page which was last edited on 31st August - *this* Tuesday, at the time of posting this message.

Wiki.js shows the last edit date as "*Last* Tuesday at ...", which is not correct. Currently, *last* Tuesday would be 24th August.

> Generally if you say "last Tuesday", people understand you to mean the Tuesday from the preceding calendar week.
> https://ell.stackexchange.com/a/15794

To avoid any possibility of confusion, it would probably be better to avoid using *"Last ..."* in the relative date formats. Use the day name for dates within the current week, and either the date or *"{x} days ago"* for dates in the previous week.

**Comment by NGPixel** [comment]
Relative dates are displayed using [Moment.js calendar time function](https://momentjs.com/docs/#/displaying/calendar-time/), which should display what you are describing. Are you sure the date isn't actually Tuesday from last week? In most cases, you can see the full date by mouse over the relative date.

**Reply by RichardD2** [reply]
I am sure - the date was Tuesday 31st August, and I was viewing the page on Friday 3rd September. There is no tooltip for the date on the page itself, only in the edit history.

![image](https://user-images.githubusercontent.com/2158265/132177239-56769275-120c-4e18-a552-158727ec3c11.png)
![image](https://user-images.githubusercontent.com/2158265/132177382-3687b7c0-b06a-456e-8291-f3b06a60089f.png)
